app/pages/sections/catalogue-solutions/order-item/controller.test.js

```diff
@@ -31,6 +40,16 @@ const selectedPrice = {
   price: 0.1,
 };
 
+const detail = {
```

Comment:
rather than declare this at the top can you just declare it in test it is being used

Reply:
done